src/components/Match/matchPages.jsx

```diff
@@ -35,6 +36,19 @@ import Chat from './Chat';
 import { StyledFlexContainer, StyledFlexElement } from './StyledMatch';
 
 const matchPages = [Overview, {
+  name: strings.tab_drafts,
+  key: 'draft',
+  content: match => (
+    <div>
+      {match.game_mode === 2 &&
+        <Draft
+          radiantTeam={match.radiant_team}
+          direTeam={match.dire_team}
+          draft={match.draft_timings}
```

Comment:
This won't exist for all matches, so ensure the component handles the cases where the draft_timings are null/undefined (unparsed matches)